_posts/2023-02-15-a-necessary-breaking-change.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "A necessary breaking change in the Regex engine"
+author: beta-ziliani
+summary: "PCRE is at EOL, we need to move to PCRE2"
+---
+
+Crystal uses since its inception the [PCRE](https://www.pcre.org/) library for dealing with regular expressions. This library has two major versions, and Crystal so far resorted to the first one (PCRE). However, this version is getting at its end of life. Therefore, for the next release (1.8) we are planning to move to its successor, PCRE2.

```

Comment:
PCRE isn't getting there, it has formally reached EOL with version 8.45 (2021-06-15).
```suggestion
Crystal uses since its inception the [PCRE](https://www.pcre.org/) library for dealing with regular expressions. This library has two major versions, and Crystal so far resorted to the first one (PCRE). However, this version has reached its end of life. Therefore, for the next release (1.8) we are planning to move to its successor, PCRE2.

```